performance_test/src/experiment_configuration/experiment_configuration.cpp

```diff
@@ -140,6 +141,13 @@ void ExperimentConfiguration::setup(int argc, char ** argv)
 
     if (vm["communication"].as<std::string>() == "ROS2") {
       m_com_mean = CommunicationMean::ROS2;
+    } else if (vm["communication"].as<std::string>() == "ROS2PollingSub") {
+#ifdef PERFORMANCE_TEST_POLLING_SUBSCRIPTION_ENABLED
+      m_com_mean = CommunicationMean::ROS2PollingSub;
+#else
+      throw std::invalid_argument(
+              "You must compile with ApexOS to enable it as communication mean.");
```

Comment:
Wrong error message. You must compile with POLLING SUB ENABLED

Reply:
Okay I will change the message appropriately.